daprdocs/content/en/developing-applications/ides/intellij.md

```diff
@@ -23,18 +23,53 @@ Let's get started!
 
 ## Add Dapr as an 'External Tool'
 
-First, quit IntelliJ.
+First, quit IntelliJ before modifying the configurations file directly.
 
-Create or edit the file in `$HOME/.IdeaIC2019.3/config/tools/External\ Tools.xml` (change IntelliJ version in path if needed) to add a new `<tool></tool>` entry:
+### IntelliJ configuration file location
+For versions [2020.1](https://www.jetbrains.com/help/idea/2020.1/tuning-the-ide.html#config-directory) and above the configuration files for tools should be located in:
+
+{{< tabs Windows Linux  MacOS >}}
+
+{{% codetab %}}
+
+```powershell
+%USERPROFILE%\AppData\Roaming\JetBrains\IntelliJIdea2020.1\tools\
+``` 
+{{% /codetab %}}
+
+
+{{% codetab %}}
+ ```shell
+ $HOME/.config/JetBrains/IntelliJIdea2020.1/tools/
+ ```
+{{% /codetab %}}
+
+
+{{% codetab %}}
+```shell
+~/Library/Application Support/JetBrains/IntelliJIdea2020.1/tools/
+``` 
+{{% /codetab %}}
+
+
+{{< /tabs >}}
+
+> The configuration file location is different for version 2019.3 or prior. See [here](https://www.jetbrains.com/help/idea/2019.3/tuning-the-ide.html#config-directory) for more details.
+
+Change the version of IntelliJ in the path if needed.
+
+Create or edit the file in `<CONFIG PATH>/tools/External\ Tools.xml` (change IntelliJ version in path if needed). The `<CONFIG PATH>` is OS dependennt as seen above.
+
+Add a new `<tool></tool>` entry:
 
 ```xml
 <toolSet name="External Tools">
   ...
   <!-- 1. Each tool has its own app-id, so create one per application to be debugged -->
   <tool name="dapr for DemoService in examples" description="Dapr sidecar" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="true" showConsoleOnStdErr="true" synchronizeAfterRun="true">
     <exec>
-      <!-- 2. For Linux or MacOS use: /usr/local/bin/daprd -->
-      <option name="COMMAND" value="C:\dapr\daprd.exe" />
+      <!-- 2. For Linux or MacOS use: /usr/local/bin/dapr -->
+      <option name="COMMAND" value="C:\dapr\dapr.exe" />
       <!-- 3. Choose app, http and grpc ports that do not conflict with other daprd command entries (placement address should not change). -->
       <option name="PARAMETERS" value="-app-id demoservice -app-port 3000 -dapr-http-port 3005 -dapr-grpc-port 52000 -placement-host-address localhost:50005" />
```

Comment:
run

Reply:
Also, you don't pass the placement host address in dapr

Reply:
Thanks. fixed it